test/data/brave-wallet/brave_wallet_event_emitter.html

```diff
@@ -3,7 +3,19 @@
 <script>
   var received_connect_event = false
   var received_chain_changed_event = false
-  console.log('page loaded!')
+  var received_account_changed_event = false
+  window.ethereum.on('connect', function(chainId) {
+    received_connect_event = true
+  });
+  window.ethereum.on('chainChanged', function(chainId) {
+    received_chain_changed_event = chainId == '0x5'
+  });
+  window.ethereum.on('accountsChanged', function(accounts) {
+    received_account_changed_event = true
+  });
+  if (window.ethereum.isConnected()) {
+    received_connect_event = true
+  }
```

Comment:
Why are we setting `received_connect_event` here? This is not a connect event

Reply:
it means that the event is already happened

Reply:
you already has connect event handler
```
 window.ethereum.on('connect', function(chainId) {
    received_connect_event = true
  });
  ```
why do we need to check isConnect again?

Reply:
ok, as you wish

Reply:
done!